### DISCUSSION TITLE
Python version > 3,8

### DISCUSSION THREAD

**Discussion by manuelchaves** [KlipperScreen/KlipperScreen#1427] [Q&A]
After  KlipperScreen update:

###### Installing KlipperScreen ...
Checking Python version > 3,8
Python 3.7.3
Not supported

MKS SKIPR V1.0 BOARD

I can't upgrade from Python 3.7.3 to 3.8 or higher, it always returns 3.7.3

**Comment by Amirkaregari** [comment]
i had same issue did you have any solution?

**Comment by thijstriemstra** [comment]
what operating system (version) are you using? you can either upgrade OS (eg. debian bookworm) or use pyenv to upgrade python only, eg: pyenv install 3.10.4

**Comment by pgale** [comment]
I have the same. Debian 10. I upgraded to Debian 11 which has python 3.9.2 but on startup, KlipperScreen still errors with "The system doesn't meed to minimum requirement" System: 3.7
But, the installed version is 3.9.2

Any ideas?

**Reply by pgale** [reply]
I resolved this by reinstalling KlipperScreen.
